be/src/exec/mor_processor.h

```diff
@@ -1,72 +0,0 @@
-// Copyright 2021-present StarRocks, Inc. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#pragma once
-
-#include <atomic>
-
-#include "exec/pipeline/hashjoin/hash_joiner_fwd.h"
-#include "exprs/expr_context.h"
-#include "runtime/descriptors.h"
-#include "util/runtime_profile.h"
-
-namespace starrocks {
-
-struct MORParams {
-    TupleDescriptor* tuple_desc = nullptr;
-    TupleDescriptor* delete_column_tuple_desc = nullptr;
-    std::vector<SlotDescriptor*> equality_slots;
-    RuntimeProfile* runtime_profile = nullptr;
-    int mor_tuple_id;
-};
-
-class DefaultMORProcessor {
```

Comment:
If I delete this, will there be any problems during grayscale upgrade?

Reply:
I have tested it.  we will throw an exception. 
ERROR 1064 (HY000): Unsupported iceberg file content: 2 in the scanner thread. 
Relatively few users of mor scene.